### ISSUE TITLE
Windows 10, task switch from Tabby to another task failed.

### ISSUE THREAD

**Issue by iJoez** [Eugeny/tabby#8300]
Version: 1.0.196
Platform: Windows x64 10.0.19044
Plugins: none
Frontend: xterm-webgl

Cannot use key ALT-TAB to switch to another task when Tabby terminal is in foreground, and Tabby still stay in front of the screen, which can be reproduced.

**Comment by Eugeny** [comment]
Did you set Tabby to "dock always on top" in Settings -> Window?

**Comment by iJoez** [comment]
Yes, you are right. I was not aware of this function. After disabling this, I can switch tasks. Thanks for response.
